### ISSUE TITLE
Node modules not installed?

### ISSUE THREAD

**Issue by ma-armenta** [RMCob/homebridge-wyze-robovac#3]
python3: can't open file '/usr/lib/node_modules/homebridge-wyze-robovac/py_helpers/getVacuumDeviceList.py': [Errno 2] No such file or directory

**Comment by RMCob** [comment]
Please uninstall and reinstall using the Homebridge GUI and report back.

**Comment by ma-armenta** [comment]
There is no option to uninstall it 
![image](https://user-images.githubusercontent.com/37154380/171211786-9362d7ec-8d68-4c0f-b37f-3c1d66f303e3.png)

**Comment by ma-armenta** [comment]
I found the issue. There is an alternative location for the python scripts.
/homebridge/node_modules/homebridge-wyze-robovac/py_helpers <- absolute path

**Comment by bobziroll** [comment]
@ma-armenta did you manage to get it to work? I updated my absolute path in the config and now it's saying there's no module named `wyze-sdk`. Did you need to manually do some kind of dependencies install? Or did it just work for you after restarting homebridge?

**Comment by alanhartless** [comment]
FWIW, I managed to get past this on a Sysology NAS although I can't say if it's the right way or not. After updating my path, I was also getting the error that wyze_sdk couldn't be found. I tried installing it system wide but then got the error that it couldn't import Client from wyze_sdk. 

I got it working by updating /etc/passwd to enable sh access to the homebridge user by chancing `/sbin/nologin` to `/bin/sh`. Restarted the NAS. Then I installed wyze_sdk with `sudo -H -u homebridge python3 -m pip install wyze_sdk` Now I'm getting the same error as https://github.com/RMCob/homebridge-wyze-robovac/issues/16 but we have progress.
